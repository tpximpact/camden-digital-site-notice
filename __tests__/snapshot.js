import { render } from "@testing-library/react";
import Home from "../src/app/(main)/page";

jest.mock("react", () => {
  const testCache = (func) => func;
  const originalModule = jest.requireActual("react");
  return {
    ...originalModule,
    cache: testCache,
  };
});

it("renders homepage unchanged", () => {
  const { container } = render(<Home />);
  expect(container).toMatchSnapshot();
});
