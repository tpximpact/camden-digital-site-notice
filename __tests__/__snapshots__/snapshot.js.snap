// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders homepage unchanged 1`] = `
<div>
  <div
    class="wrap-home"
  >
    <h1
      class="govuk-heading-xl"
      role="heading"
      style="display: inline-block;"
    >
      Find planning applications near you
    </h1>
    <p
      class="govuk-body-m"
    >
      Find, review and leave your comments on planning applications in
       
    </p>
    <section
      class="search-grid"
    >
      <div
        class="govuk-form-group false"
      >
        <h1
          class="govuk-label-wrapper"
        >
          <label
            class="govuk-label govuk-label--l false"
            for="event-name"
            role="definition"
          >
            Enter a postcode to find planning applications nearby
          </label>
        </h1>
        <input
          class="govuk-input false"
          id="event-name"
          name="eventName"
          role="textbox"
          type="text"
          value=""
        />
      </div>
      <button
        class="govuk-button grid-button-search"
        data-module="govuk-button"
      >
        Search
        <span
          class="undefined"
          style="margin-left: 10px; vertical-align: middle;"
        >
          <svg
            fill="none"
            height="19"
            viewBox="0 0 19 19"
            width="19"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              clip-rule="evenodd"
              d="M0.772461 0L10.8382 9.5L0.772461 19H8.20461L16.5662 11.1083L18.2703 9.5L16.5662 7.89172L8.20461 0H0.772461Z"
              fill="white"
              fill-rule="evenodd"
            />
          </svg>
        </span>
      </button>
    </section>
    <div
      class="wrap-pagination"
    >
      <ul
        aria-label="Pagination"
        class="pagination govuk-body"
        role="navigation"
      >
        <li
          class="page-item disabled"
        >
          <a
            aria-disabled="true"
            aria-label="Previous page"
            class="page-link "
            rel="prev"
            role="button"
            tabindex="-1"
          >
            <svg
              fill="none"
              height="13"
              viewBox="0 0 15 13"
              width="15"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M6.893 13.125L8.3066 11.711L4.014 7.418L17 7.418V5.418L4.104 5.418L8.2895 1.4414L6.9125 -0.0078001L0.1684 6.3984L6.893 13.125Z"
                fill="#505A5F"
              />
            </svg>
          </a>
        </li>
        <li
          class="page-item"
        >
          <a
            aria-disabled="false"
            aria-label="Next page"
            class="page-link"
            rel="next"
            role="button"
            tabindex="0"
          >
            <svg
              fill="none"
              height="13"
              viewBox="0 0 15 13"
              width="15"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M8.107 -0.0078125L6.6934 1.40619L10.986 5.69919H-2V7.69919H10.896L6.7105 11.6758L8.0875 13.125L14.8316 6.71879L8.107 -0.0078125Z"
                fill="#505A5F"
              />
            </svg>
          </a>
        </li>
      </ul>
    </div>
  </div>
</div>
`;
